apps/modernization-api/src/test/resources/application-test.yml

```diff
@@ -32,7 +32,7 @@ spring:
         hbm2ddl:
           jdbc_metadata_extraction_strategy: individually
   datasource:
-    url: jdbc:sqlserver://${nbs.datasource.server}:1433;database=nbs_odse;encrypt=true;trustServerCertificate=true;
+    url: jdbc:sqlserver://${nbs.datasource.server}:1434;database=nbs_odse;encrypt=true;trustServerCertificate=true;
```

Comment:
`nbs.datasource.port` property maybe?  That was we can override it locally if we want to run tests against the local development database.  We may have to specify this value like we do with the elasticsearch test-container